tests/index.test.ts

```diff
@@ -103,3 +104,162 @@ test('with harmony', async () => {
 
   process.version = _version
 })
+
+describe('modern ES_VERSION without harmony', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line no-extra-parens
+    (fetch as any as jest.Mock).mockResolvedValue({
+      json() {
+        return Promise.resolve(fixture.modern)
+      },
+    })
+  })
+
+  test('ES2022', async () => {
```

Comment:
There's no need to add every years.

Reply:
I didn't add every year. I added 2 cases.

Reply:
I think one year is enough.

Reply:
You're welcome to push changes directly to the branch if you want to reduce the scope of the tests.

Reply:
I'm busy, and you're working on this pull request.

Reply:
I'm busy too.. The PR is perfectly valid with 2 years in the test cases. You didn't like that.. so you want me to put in extra work to align your stylistic choices. You could even do the whole thing through the UI and make a suggested change to remove it like you did before and then it would be a single button click for either of us to resolve, but you don't..?

Reply:
Fixed with:

- `add missing test fixture` [`6b433db` (#3)](https://github.com/g-plane/node-green/pull/3/commits/6b433dba47d54432e78abe7f0e52092d841ef9ac)
- `reduce test reliability to satisfy code review` [`c37a10f` (#3)](https://github.com/g-plane/node-green/pull/3/commits/c37a10f051ca6f47e56fe26c144ce5d81374e41b)